README.md

```diff
@@ -59,6 +59,22 @@ ROTEL_OTLP_EXPORTER_CUSTOM_HEADERS="Authorization=Bearer ${AXIOM_API_KEY},X-Axio
 The values `${AXIOM_API_KEY}` and `${AXIOM_DATASET}` will be resolved from the environment of the function,
 allowing you to set the secret values in your AWS Lambda function definition and out of the on-disk file.
 
+### Secrets
+
+Secret values can be retrieved from [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/) by specifying the full
```

Comment:
Did you want to add something here about the Secrets Manager API latency impact?